src/types/clip.ts

```diff
@@ -0,0 +1,10 @@
+export type TClip = {
+	_id: string;
+	title: string;
+	content?: string; // OPTIONAL BUT ONLY TEMPORARILY
```

Comment:
you can put this on required. Cuz this wont have merge conflicts.
```suggestion
	content: string;
```